moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryCouncilDistricts.js

```diff
@@ -11,9 +11,15 @@ const getCouncilDistricts = (data) => {
 
   // flatten the array of arrays and remove empty districts
   const districtsArray = districts.flat().filter((d) => d);
+  return districtsArray;
+};
 
+const getAllCouncilDistricts = (projectGeography, childProjectGeography) => {
```

Comment:
ahh cool - i like how this code spells out what makes up the council districts (parent + child) 🚀